src/NServiceBus.AcceptanceTesting/Customization/BusConfigExtensions.cs

```diff
@@ -0,0 +1,16 @@
+namespace NServiceBus.AcceptanceTesting.Customization
+{
+    using System;
+    using System.Collections.Generic;
+
+    public static class BusConfigExtensions
+    {
+        /// <summary>
+        /// Backdoor into the core types to scan. This allows people to test a subset of functionality when running Acceptance tests
```

Comment:
Not sure I totally understand the question, anyway I'll try to answer it.
This is a hack to support our AT framework.
Users should not require this hack.